app/views/ShareView/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Image as ImageCompressor, Video as CompressVideo } from 'react-native-compressor';
+
+export type TQuality = 'SD' | 'HD';
+export const QUALITY_SD = 0.5;
+
+export const compressImage = async (uri: string, quality: TQuality) => {
```

Comment:
You're only calling `compressImage` and `compressVideo` to compress them.
No point on receiving `quality` as a param.

```suggestion
export const compressImage = async (uri: string) => {
```